src/odemis/driver/technolution.py

```diff
@@ -443,18 +533,221 @@ def _mergeMetadata(self):
         return md
 
     def _setAcqDelay(self, delay):
+        """
+        Setter which checks if detector can record images before ebeam scanner has started to scan.
+
+        :param delay (tuple):
+        :return (tuple):
+        """
         # Check if detector can record images before ebeam scanner has started to scan.
-        if delay - self.parent._ebeam_scanner.scanDelay.value[0] >= 0:
+        if delay >= self.parent._ebeam_scanner.scanDelay.value[0]:
             return delay
         else:
             # Change values so that 'self.acqDelay.value - self.parent._ebeam_scanner.scanDelay.value[0]' has a positive result
```

Comment:
This comment is cryptic. Rephrase in words what you do. Don't repeat code in comments or doc strings.
You return the old value on the VA, if the new requested value does not fit the requirements that acqDelay needs to be larger than scanDelay?